docs/scenarios/aistudio/aistudio-testing.md

```diff
@@ -0,0 +1,25 @@
+# Testing scenarios using AI Studio extension
+
+## Prerequisities
+1. Podman installed
+2. Machine created with at least 2 cores and 10GB of memory
+
+## Instalation and starting up of AI Studio extension
+
+1. Open Podman Desktop Settings -> Extensions
+2. Install a new extension from OCI Image -> `ghcr.io/projectatomic/ai-studio:latest` (possible to also test `:nightly` for latest builds - every commit from `https://github.com/projectatomic/ai-studio`)
+3. Verify that AI Studio extension is present under Settings -> Extensions
+4. Verify that Studio extension is present in the left navbar
+
+## Running an AI Studio demo
+
+1. Click on Studio extension in the left navbar
+2. Click on Recipe Catalog
+3. Click on ChatBot demo application
+4. Install demo application - will download a rather large LLM model from HuggingFace
```

Comment:
Start AI app is the option, right? There are two models that can be picked. I would also suggest to switch the tab and verify the default model is picked. we can be also specific about the model to use.

Reply:
I mean, both models are from huggingFace.

Reply:
fixed